### ISSUE TITLE
Keyword search - returning wrong results

### ISSUE THREAD

**Issue by julie-sullivan** [alexkalderimis/staircase#49]
If I search FlyMine for `kinase*`, I get these types:

```
Hits by Category
Protein: 69247
Gene: 30928
Publication: 9302
Uniprot Feature: 2188
Protein Domain: 1502
Interaction Experiment: 1222
GO Term: 708
Component: 47
Pathway: 44
Ontology Term: 19
Interaction Term: 7
Disease: 6
cDNA Clone: 1
```

When I search for the same term in stepz I only get 7 categories:

![kinase](https://cloud.githubusercontent.com/assets/1663057/3599120/3e2c19c6-0ceb-11e4-8d11-c962b474de3e.png)

It looks like we are just getting the first 100 results instead of getting all the results. These truncated results are wrong, it looks like we don't have any kinase GO terms or protein domains.

**Comment by julie-sullivan** [comment]
@sergiocontrino Do you think this would have been fixed by 1.4?

**Comment by sergiocontrino** [comment]
yes @julie-sullivan, this is fixed in 1.4 (for the webapp, when creating a list from a search results). and keyword results page is fine.

**Comment by julie-sullivan** [comment]
@sergiocontrino I suppose my question is would that fix fix it in staircase too?
